queries.sql

```diff
@@ -215,4 +215,6 @@ JOIN vets ON visits.vet_id = vets.id
 WHERE vets.name = 'Maisy Smith'
 GROUP BY species.name
 ORDER BY num_visits DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
+SELECT COUNT(animal_id) FROM visits WHERE animal_id = 4;
```

Comment:
- [ ] As stated in the project requirement **you were challenged with 3 queries that take too much time** (check project description for more details):
- **Use EXPLAIN ANALYZE on the previous queries to check what is happening. Take screenshots of them** .Find a way to decrease the execution time of the first query and Find a way to improve execution time of the other two queries.

- While you are making changes, check if they help by running **EXPLAIN ANALYZE. Once you succeed, take a screenshot of the EXPLAIN ANALYZE result showing that time actually decreased.**
- But In the case of yours you only check the first query so kindly make sure that you also check the remaining two query.

```
SELECT COUNT(*) FROM visits where animal_id = 4;
SELECT * FROM visits where vet_id = 2;
SELECT * FROM owners where email = 'owner_18327@mail.com'; 
```